components/select/nz-select.component.ts

```diff
@@ -154,6 +154,7 @@ export class NzSelectComponent implements ControlValueAccessor, OnInit, AfterVie
   /** https://github.com/angular/angular/pull/13349/files **/
     // tslint:disable-next-line:no-any
   @Input() compareWith = (o1: any, o2: any) => o1 === o2;
+  @Input() tabindex = 1;
```

Comment:
The default value of `tabindex` attribute in [HTML Living Standard](https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex) is 0 and I think we should align with that.

Also note that it is [not recommended](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) to use positive tabindex in most cases.

Reply:
yes,the default value shoud be 0,but sometimes it is necessary to skip it.